DEVELOPING_OPAL/demos/src/main/scala/org/opalj/ai/ExceptionUsage.scala

```diff
@@ -78,64 +76,53 @@ object ExceptionUsage extends ProjectAnalysisApplication {
                 val operands = result.operandsArray(pc)
                 if (operands != null) { // the instruction is reached...
                     instruction match {
-                        case ATHROW =>
-                            updateUsageKind(operands.head, UsageKind.IsThrown)
+                        case ATHROW => updateUsageKind(operands.head, UsageKind.IsThrown)
                         case i: MethodInvocationInstruction =>
                             val methodDescriptor = i.methodDescriptor
-                            val parametersCount = methodDescriptor.parametersCount
+                            val parametersCount  = methodDescriptor.parametersCount
                             operands.take(parametersCount).foreach(updateUsageKind(_, UsageKind.UsedAsParameter))
                             if (i.isVirtualMethodCall) {
                                 updateUsageKind(operands.drop(parametersCount).head, UsageKind.UsedAsReceiver)
                             }
-                        case _: FieldWriteAccess =>
-                            updateUsageKind(operands.head, UsageKind.StoredInField)
-                        case ARETURN =>
-                            updateUsageKind(operands.head, UsageKind.IsReturned)
-                        case AASTORE =>
-                            updateUsageKind(operands.head, UsageKind.StoredInArray)
-                        case _ =>
+                        case _: FieldWriteAccess => updateUsageKind(operands.head, UsageKind.StoredInField)
+                        case ARETURN             => updateUsageKind(operands.head, UsageKind.IsReturned)
+                        case AASTORE             => updateUsageKind(operands.head, UsageKind.StoredInArray)
+                        case _                   =>
                         /*nothing to do*/
                     }
                 }
             }
 
             val usages =
-                for { ((pc, typeName), exceptionUsage) <- exceptionUsages }
-                    yield ExceptionUsage(method, pc, typeName, exceptionUsage)
+                for { ((pc, typeName), exceptionUsage) <- exceptionUsages } yield ExceptionUsage(method,
+                                                                                                 pc,
+                                                                                                 typeName,
+                                                                                                 exceptionUsage)
 
-            if (usages.isEmpty)
-                None
+            if (usages.isEmpty) None
             else {
                 Some(usages)
             }
         }).filter(_.isDefined).map(_.get).flatten
 
         val (notUsed, used) = usages.toSeq.partition(_.usageInformation.isEmpty)
-        var report = used.map(_.toString).toList.sorted.mkString("\nUsed\n", "\n", "\n")
+        var report          = used.map(_.toString).toList.sorted.mkString("\nUsed\n", "\n", "\n")
```

Comment:
Why does it align regular local variable definitions? Do we really want this?

Reply:
we have to try with `align.preset = some` and we what the impact of that is